dev.yml

```diff
@@ -0,0 +1,12 @@
+---
+name: kubernetes-deploy
+up:
+  - ruby: 2.4.1
```

Comment:
Let's use 2.3.3 instead to match Shipit (see also https://github.com/Shopify/kubernetes-deploy/issues/134)